black-jack-redo.rb

```diff
@@ -25,88 +25,205 @@ def print_with_stars(string)
 	puts string
 	puts '*' * 60
 	puts '*' * 60
+	puts '--------'
 end
 
-def viewhand(array, string)
-	count3 = 0
-	count4 = 0
-	while count3 < array.length
-		if array[count3][:dealt] == true
-			string += array[count3][:card] + ' of ' + array[count3][:suit] + "\n"
-			count4 += array[count3][:value]
-		else
-		end
-		count3 += 1
+def get_suit(string)
+	card_suit = ''
+	if string[0] == 'S'
+		card_suit = "Spades\n"
+	elsif string[0] == 'H'
+		card_suit = "Hearts\n"
+	elsif string[0] == 'D'
+		card_suit = "Diamonds\n"
+	elsif string[0] == 'C'
+		card_suit = "Clubs\n"
+	else 
+		puts 'Something went wrong!'
 	end
-	return count4, string
+
+	return card_suit
+end
+
+def get_card(card)
+	user_card = ''
```

Comment:
This is a bad name for a variable. Try to think of a better one that communicates what this is doing.